<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StagedCelWidget</class>
 <widget class="QWidget" name="StagedCelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="thumbLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>#IMAGE</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="infoAndOptionsLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>#NAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="geometryLabel">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>#GEOMETRY</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="optionButtons">
         <item>
          <widget class="QToolButton" name="showCelInfoButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../blit.qrc">
             <normaloff>:/info_icon</normaloff>:/info_icon</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="moveLayerButtons" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QToolButton" name="raiseButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../blit.qrc">
           <normaloff>:/arrow_up_icon</normaloff>:/arrow_up_icon</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="lowerButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../blit.qrc">
           <normaloff>:/arrow_down_icon</normaloff>:/arrow_down_icon</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../blit.qrc"/>
 </resources>
 <connections/>
</ui>
